README.md

```diff
@@ -2,95 +2,147 @@
 
 [![GitHub Action](https://github.com/datastax/cql-proxy/actions/workflows/test.yml/badge.svg)](https://github.com/datastax/cql-proxy/actions/workflows/test.yml) [![Go Report Card](https://goreportcard.com/badge/github.com/datastax/cql-proxy)](https://goreportcard.com/report/github.com/datastax/cql-proxy)
 
-A CQL proxy/sidecar. It listens on a local address and securely forwards your application's CQL traffic.
+## Table of Contents
 
-**Note**: `cql-proxy` in its current state works well, but it is still under development. That means
-that things might break or change. Please give it a try and let us know what you think!
+- [What is the cql-proxy?](https://github.com/datastax/cql-proxy#what-is-cqlproxy)
+- [When to use cql-proxy](https://github.com/datastax/cql-proxy#when-to-use-cql-proxy)
+- [Getting started](https://github.com/datastax/cql-proxy#getting-started)
+- [Configuration](https://github.com/datastax/cql-proxy#configuration)
+
+## What is cql-proxy?
 
 ![cql-proxy](cql-proxy.png)
 
+`cql-proxy` is designed to forward your application's CQL traffic to an appropriate database service. It listens on a local address and securely forwards that traffic.
+
+**Warning**: `cql-proxy` in its current state works well, and you should give it a try. However, it is still under development, so things might break or change. 
+
+Please give it a try and let us know what you think!
+## When to use `cql-proxy`
+
+The `cql-proxy` sidecar enables unsupported CQL drivers to work with [DataStax Astra][astra]. These drivers include both legacy DataStax [drivers] and community-maintained CQL drivers, such as the [gocql] driver and the [rust-driver].
+
+`cql-proxy` also enables applications that are currently using [Apache Cassandra][cassandra] or [DataStax Enterprise (DSE)][dse] to use Astra without requiring any code changes.  Your application just needs to be configured to use the proxy.
+
+If you're building a new application using DataStax [drivers], `cql-proxy` is not required, as the drivers can communicate directly with Astra. DataStax drivers have excellent support for Astra out-of-the-box, and are well-documented in the [driver-guide] guide. 
 ## Getting started
 
-```sh
-go build
-```
+There are three methods for using `cql-proxy`:
 
-Run against a [DataStax Astra][astra] cluster:
+- Locally build and run `cql-proxy`
+- Run a docker image that has `cql-proxy` installed
+- Use k8ssandra to run `cql-proxy` in a Kubernetes container 
```

Comment:
k8ssandra is not needed. `cql-proxy` would be deploy in their k8s cluster as a sidecar in there applications pod.

Reply:
Yup - pulled out.